test/OpenTelemetry.ClrProfiler.Managed.Bootstrapping.Tests/InstrumentationTests.cs

```diff
@@ -0,0 +1,75 @@
+using System;
+using System.Diagnostics;
+using OpenTelemetry.Trace;
+using Xunit;
+
+namespace OpenTelemetry.ClrProfiler.Managed.Bootstrapping.Tests
+{
+    public class InstrumentationTests
+    {
+        private readonly ActivitySource _otelActivitySource = new("OpenTelemetry.ClrProfiler.*");
+        private readonly ActivitySource _customActivitySource = new("Custom");
+
+        public InstrumentationTests()
+        {
+            // Exporter should always be set to a valid value
+            Environment.SetEnvironmentVariable("OTEL_EXPORTER", "zipkin");
```

Comment:
We should rather support a scenario where the exporter is not set via env var. One can always use a plugin to load it. Can you create an issue for it and reference it in this comment?

Reply:
This is a very simple fix, I will just include it here